### PR TITLE
Add trigger support

### PR DESCRIPTION
**Pull request by g-despot**

### CODE REVIEW

File: gqlalchemy/models.py [hunk 1]
```diff
@@ -248,9 +281,5 @@ def __init__(self, **data):
 
     def __str__(self) -> str:
         return "".join(
-            (
-                f"<{type(self).__name__}",
-                f" nodes={self._nodes}",
-                f" relationships={self._relationships}" ">",
-            )
+            (f"<{type(self).__name__}", f" nodes={self._nodes}", f" relationships={self._relationships}" ">",)
         )
```
Comment (MasterMedo) [inline gqlalchemy/models.py:285]: Here is a mistype, can you find it? :D
p.s.
Not made by you.

File: tests/intergration/test_trigger.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    )
+
+    memgraph.create_trigger(trigger)
+    assert any(map(lambda t: t["trigger name"] == "test_trigger", memgraph.get_triggers()))
+
+
+def test_drop_trigger(memgraph: Memgraph):
+    trigger = MemgraphTrigger(
+        name="test_trigger",
+        event_type=TriggerEventType.CREATE,
```
Comment (MasterMedo) [inline tests/intergration/test_trigger.py:43]: Shouldn't here be `DELETE` trigger event type?

File: gqlalchemy/memgraph.py [hunk 3]
```diff
@@ -149,6 +151,25 @@ def drop_database(self):
         """Drops database by removing all nodes and edges"""
         self.execute("MATCH (n) DETACH DELETE n;")
 
+    def create_trigger(self, trigger: MemgraphTrigger):
+        """Creates a trigger"""
+        print(trigger)
```
Comment (gitbuda) [inline gqlalchemy/memgraph.py:156]: Remove this prints 😄

Comment (MasterMedo) [reply gqlalchemy/memgraph.py]: done

**Review by MasterMedo — APPROVED** [review]

**Pull request merged**